pkg/rslog/logger.go

```diff
@@ -97,6 +102,18 @@ func (o *LogLevel) UnmarshalText(text []byte) (err error) {
 	return fmt.Errorf("invalid Log Level value '%s'. Allowed values are %v", text, values)
 }
 
+func (o *LogLevel) Compare(level LogLevel) int {
```

Comment:
@marcosnav @eduardocfalcao I just added this method this morning so we can check for a compatible log level in Connect when debug regions are enabled. I also added a corresponding test in `logger_test.go`.